README.md

```diff
@@ -37,6 +37,7 @@ This library currently supports the following cryptocurrencies and address forma
  - ETC (checksummed-hex)
  - RSK (checksummed-hex)
  - XDAI (checksummed-hex)
+ - CELO (checksummed-hex)
```

Comment:
Can you provide reference link so that we can verify?

Reply:
@makoto https://docs.celo.org/overview#background-and-key-concepts